Octokit.Tests/Helpers/Arg.cs

```diff
@@ -53,6 +53,21 @@ public static NewRepository NewRepository
             get { return Arg.Is<Dictionary<string, string>>(d => d.Count == 0); }
         }
 
+        public static Dictionary<string, string> DictionaryWithSince
```

Comment:
I like `DictionaryWithApiOptions` for the general case, but these feel very specific to gists.

Should these live here, or do you think they should live closer to the tests themselves?